src/physics/utilities/solver_config.hpp

```diff
@@ -20,6 +20,7 @@ namespace serac {
 enum class OutputType
 {
   GLVis,
+  ParaView,
```

Comment:
A test for this output type is needed.

Reply:
Yes, which example would be good to change the output type on?

Reply:
Hmmm, really any of them is fine. Maybe one of the glvis ones (thermal-explicit?) to help clean out the output in the test directory?